client/styles/styles.css

```diff
@@ -270,6 +270,92 @@ body {
   color: #ffffff;
 }
 
+/* CAROUSEL-COMPONENT */
+
+.homepage__carousel {
+  background-color: #ffffff;
+  text-align: center;
+}
+
+.carousel-inner {
+  border-radius: 20px 20px 0 0;
+}
+
+.carousel__title {
+  font-family: Poppins;
+  text-align: center;
+  font-size: 36px;
+  font-weight: bolder;
+  color: #1C2294;
+  margin-top:20px;
+}
+
+.carousel__subtittle {
+  text-align: center;
+  font-family: Poppins;
+  font-size: 48px;
+  font-weight: bolder;
+  color: #1C2294;
+  margin-bottom: 30px;;
+}
+
+.container {
```

Comment:
To avoid conflicts with other styling I would rename this to
`.homepage-carousel-container`
**Don't forget to update in component as well !**

Reply:
Ok

Reply:
Another mystery: Yesterday I did not like the name of this class, but if I changed it the carousel size increases even more and I don't want that.
I tried yesterday too. (Note: I tried with different names, and did it in styles.css file and in my component).
You can try too. I definitely thought it was related with bootstrap classes.